bors.toml

```diff
@@ -5,3 +5,4 @@ status = [
 delete_merged_branches = true
 timeout_sec = 86400
 block_labels = [ "do-not-merge-yet" ]
+cut_body_after = " <!--"
```

Comment:
Won't this just cut the entire PR discussion because the very first thing in the template?

I am happy to remove this from the template if that's what everyone wants. I really don't understand why bors pastes the PR discussion into the commit in the first place.

Reply:
It doesn't put the whole discussion in, just the merge commit message is the text in the PR description.

Reply:
Also I seem to have a spurious " " there.